### ISSUE TITLE
Support for other cascades

### ISSUE THREAD

**Issue by RTodorov** [thumbor/thumbor#481]
Hi there,

Is there any reason for Thumbor not supporting other kinds of opencv cascades? The face detection is not working for some very clear pictures and I'm about to do some experiments with other cascades like eye, fullbody, nose etc.

Do you have any interest on supporting other opencv cascades on Thumbor? If so, I may do a PR if it prooves to work better than the default cascades (default, alt, alt2, alt_tree).

BTW, shouldn't face detection work for images like these:

http://i.imgur.com/3ClzHPb.jpg
http://i.imgur.com/H0z3JYA.jpg
http://i.imgur.com/4rZfNBq.jpg

These pictures seems pretty clear to me.

I tried face detection, profile detection and glass detection, none of them worked on these 3 photos.

Thanks

**Comment by heynemann** [comment]
You can specify your own cascade file using `FACE_DETECTOR_CASCADE_FILE` (https://github.com/thumbor/thumbor/wiki/Configuration#face_detector_cascade_file).

Is that what you are looking for?

**Comment by heynemann** [comment]
As for whether those images should be detected, can you produce a python script using openCV that detects faces there? If so, then, yes, thumbor should be able to detect it (provided you use the same cascade file).

**Comment by RTodorov** [comment]
Hi @heynemann, the problem is definitely not with Thumbor, I'm just saying that opencv face detection with the 4 cascade files included in Thumbor is not working well for me. Since Thumbor only include 4 cascade files (default, alt, alt2, tree), I was wondering if any of the other cascade files that comes with opencv (for example: fullbody, mouth, nose, lowerbody etc) could do a better job than these default ones. 

But, for the tests I ran yesterday, none of them seems to work fine with my pictures, unfortunately. I really don't understand why but this is clearly a problem with opencv.

I opened this issue to discuss if it's worth to include all the other cascade files provided by opencv but apparently it's not worth of....

Here is the list of cascade files from opencv:

haarcascade_eye.xml                        
haarcascade_eye_tree_eyeglasses.xml        
haarcascade_frontalface_alt.xml            
haarcascade_frontalface_alt2.xml           
haarcascade_frontalface_alt_tree.xml       
haarcascade_frontalface_default.xml        
haarcascade_fullbody.xml                   
haarcascade_lefteye_2splits.xml            
haarcascade_licence_plate_rus_16stages.xml 
haarcascade_lowerbody.xml                  
haarcascade_mcs_eyepair_big.xml            
haarcascade_mcs_eyepair_small.xml          
haarcascade_mcs_leftear.xml                
haarcascade_mcs_lefteye.xml                
haarcascade_mcs_mouth.xml                  
haarcascade_mcs_nose.xml                   
haarcascade_mcs_rightear.xml               
haarcascade_mcs_righteye.xml               
haarcascade_mcs_upperbody.xml              
haarcascade_profileface.xml
haarcascade_righteye_2splits.xml
haarcascade_russian_plate_number.xml
haarcascade_smile.xml
haarcascade_upperbody.xml

I tried every single one of them, no lucky at all....
